drivers/hmis/app/graphql/types/hmis_schema/client_alert.rb

```diff
@@ -0,0 +1,19 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::ClientAlert < Types::BaseObject
+    description 'Alert'
+    field :id, ID, null: false
+    field :note, String, null: false
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :created_by, Types::Application::User, null: true
+    field :expiration_date, GraphQL::Types::ISO8601Date, null: true
+    field :priority, HmisSchema::Enums::ClientAlertPriorityLevel, null: false, default_value: 'low'
```

Comment:
Here, instead of `default_value: 'low'`, do you know if there is any way to refer to a value in the enum (the way you can do in java/python)? Something like `Hmis::AlertPriority::PRIORITY_LEVELS.LOW`?

Reply:
You could set it up like
```
class Hmis::ClientAlert < Hmis::HmisBase
  acts_as_paranoid

  HIGH = 'high'.freeze
  MEDIUM = 'medium'.freeze
  LOW = 'low'.freeze
  PRIORITY_LEVELS = [LOW, MEDIUM, HIGH].freeze

```

Then you would be able to access `Hmis::ClientAlert::LOW` directly.

Reply:
I like it, thanks for the suggestion!